### ISSUE TITLE
[BUG] :Unable to load callops module

### ISSUE THREAD

**Issue by eliakimfelmorais** [OpenSIPS/opensips#2139]
```
[root@centos callops]# opensips -V
version: opensips 3.1.0-beta (x86_64/linux)
flags: STATS: On, DISABLE_NAGLE, USE_MCAST, SHM_MMAP, PKG_MALLOC, Q_MALLOC, F_MALLOC, HP_MALLOC, DBG_MALLOC, CC_O0, FAST_LOCK-ADAPTIVE_WAIT
ADAPTIVE_WAIT_LOOPS=1024, MAX_RECV_BUFFER_SIZE 262144, MAX_LISTEN 16, MAX_URI_SIZE 1024, BUF_SIZE 65535
poll method support: poll, epoll, sigio_rt, select.
git revision: 93798a0
main.c compiled on 23:31:18 May 30 2020 with gcc 4.8.5
[root@centos callops]#
```

I'm Trying to load callops module but without sucess. I have followed the documentation https://opensips.org/docs/modules/3.1.x/callops.html#func_call_blind_replace but i got this error.

**To Reproduce**

I added these lines in my script and loaded the module callops.so

modparam("callops", "mode", "manual")
modparam("callops", "match_param", "call")

When I execute opensips -c 

```
[root@centos callops]# opensips -c
Jun 12 02:50:06 [30127] Traceback (last included file at the bottom):
Jun 12 02:50:06 [30127]  0. //etc/opensips/opensips.cfg
Jun 12 02:50:06 [30127] CRITICAL:core:yyerror: parse error in //etc/opensips/opensips.cfg:89:18-24: syntax error
Jun 12 02:50:06 [30127]
Jun 12 02:50:06 [30127]
Jun 12 02:50:06 [30127] modparam("callops", "match_param", manual)
Jun 12 02:50:06 [30127] ^~~~~~~
Jun 12 02:50:06 [30127] modparam("callops", "match_param", "call")
Jun 12 02:50:06 [30127] #
Jun 12 02:50:06 [30127] CRITICAL:core:yyerror: parse error in //etc/opensips/opensips.cfg:89:18-24: Invalid arguments
Jun 12 02:50:06 [30127] CRITICAL:core:yyerror: parse error in //etc/opensips/opensips.cfg:89:24-25:
Jun 12 02:50:06 [30127] ERROR:core:parse_opensips_cfg: bad config file (3 errors)
Jun 12 02:50:06 [30127] ERROR:core:main: failed to parse config file (null)
Jun 12 02:50:06 [30127] NOTICE:core:main: Exiting....
[root@centos callops]#


**OS/environment information**
 - Operating System:  CentOS Linux release 7.6.1810 (Core)


**Additional context**
<!-- Add any other context about the problem here. -->

**Comment by razvancrainea** [comment]
The logs you have sent indicate that you are missing the quotes on the `manual` param, i.e. `modparam("callops", "match_param", "manual")`. The second option is broken as well, as the value should be `"callid"` instead of `"call"`. Please try any of these two versions and let me know if it doesn't work:
`modparam("callops", "match_param", "manual")`
`modparam("callops", "match_param", "callid")`

**Comment by eliakimfelmorais** [comment]
Hi Razvan, 

Thank you. 

modparam("callops", "match_param", "manual") not work

modparam("callops", "match_param", "callid") works

**Comment by razvancrainea** [comment]
Actually, there does seem to be a bug for this, but I couldn't notice it in your logs - checking it now.

**Comment by razvancrainea** [comment]
Just committed the fix on master - if you could test, that would be great. Will soon push a fix on 3.1 as well.
